log-pipeline.yml

```diff
@@ -65,10 +76,33 @@ services:
 
   log-api:
     image: monasca/log-api:${MON_LOG_API_VERSION}
+    environment:
+      LOGSPOUT: ignore
+      LOGSTASH_FIELDS: "service=monasca-log-api"
     depends_on:
       - keystone
       - zookeeper
       - kafka
       - memcached
     ports:
       - "5607:5607"
+
+  log-agent:
+    image: monasca/log-agent:latest
+    environment:
+      LOGSPOUT: ignore
+      LOGSTASH_FIELDS: "service=monasca-log-agent"
+    depends_on:
+      - log-api
+      - keystone
+    ports:
+      - "5610:5610"
+  logspout:
+    image: monasca/logspout:latest
+    environment:
+      LOGSPOUT: ignore
+      LOGSTASH_FIELDS: "service=logspout"
```

Comment:
Same comment here - do we really want to have `LOGSTASH_FIELDS` if we ignore `logspout`